### PR TITLE
backport: bitcoin#18493,18467,18633,18157,18774,19006,18612,20033, 15283

### PR DESCRIPTION
**Pull request by vijaydasmp**

### CODE REVIEW

**Comment by vijaydasmp** [comment]
Hi @UdjinM6

File: test/functional/wallet_upgradewallet.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""upgradewallet RPC functional test
+
+Test upgradewallet RPC. Download v0.15.2 v0.16.3 node binaries:
```
Comment (UdjinM6) [inline test/functional/wallet_upgradewallet.py:7]: 18774: nit: This test should be dashified sometimes later, for now it's just skipped so it's ok-ish to have it as is probably.

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK with one nit

**Comment by vijaydasmp** [comment]
Hi @PastaPastaPasta

File: test/functional/wallet_upgradewallet.py (hunk 1 above)
Comment (vijaydasmp) [reply test/functional/wallet_upgradewallet.py:7]: Thanks , so just having as it as is

**Comment by vijaydasmp** [comment]
Hi @PastaPastaPasta requesting a review

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Pull request merged**